lib/datasource/docker/__snapshots__/index.spec.ts.snap

```diff
@@ -14,7 +14,6 @@ Array [
   },
   Object {
     "headers": Object {
-      "accept": "application/json",
```

Comment:
I'll check that, as documented [here](https://docs.docker.com/registry/spec/api/#api-version-check) the header isn't necessary.

I think we previously send it because we call `getJson` which calls got with `{json:true}` and got v9 adds the `accept` header if it doesn't exist.